docs/docs-content/release-notes/release-notes.md

```diff
@@ -11,15 +11,22 @@ tags: ["release-notes"]
 
 <ReleaseNotesVersions />
 
-## May 20, 2025 - Release 4.6.26
+## May 23, 2025 - Release 4.6.x
 
 ### Bug Fixes
 
-- Fixed an issue that occasionally caused `stylus-agent` restarts and a `stylus-operator` failure loop when deploying
+- Fixed an issue where certain artifacts were not found while syncing registries for [Edge](../clusters/edge/edge.md)
```

Comment:
```suggestion
- Fixed an issue where add-on packs/helm-chart artifacts were not found while syncing registries for [Edge](../clusters/edge/edge.md)
```
Can we make this more precise?
add-on packs/charts are not synced in this bug
whereas add-on container images are loaded properly